reference/cookie_consent.yml

```
@@ -0,0 +1,94 @@
+openapi: 3.0.1
+servers:
+  - url: 'https://{$$.env.store_domain}/api/storefront'
+info:
+  title: Storefront Cookie Consent API
+  description: Storefront API for specifying consent to particular categories of cookies for compliance purposes.
+  version: Storefront
+tags:
+  - name: Customers
+paths:
+  /consent:
+    post:
+      summary: Specify Shopper Cookie Consent Preferences
```

Update operation description to match the style of the API Reference documentation.
```suggestion
      summary: Set Cookie Consent Preferences
```